packages/vitest/src/node/config/resolveConfig.ts

```diff
@@ -160,6 +160,7 @@ export function resolveConfig(
   }
 
   resolved.clearScreen = resolved.clearScreen ?? viteConfig.clearScreen ?? true
+  resolved.configLoader = resolved.configLoader ?? viteConfig.inlineConfig.configLoader ?? 'bundle'
```

Comment:
I am actually unsure about this part. Not sure if this is needed?

Reply:
As I commented in https://github.com/vitest-dev/vitest/pull/7574#pullrequestreview-2652940426, `configLoader` is something meaningful before resolving config, so it's likely we don't need to care about and thus should be removed.